.github/workflows/pr-automation.yml

```diff
@@ -174,7 +176,8 @@ jobs:
           PW_RETRIES: ${{ vars.PW_RETRIES }}
 
   merge-reports:
-    if: "!cancelled()"
+    if: "!cancelled() && contains(github.event.pull_request.labels.*.name, 'run pw-e2e')"
```

Comment:
is it needed? there is "needs" that points to "run-tests", so the constraint to run this job if "run-tests" is finished is already here

Reply:
Fixed, you're right.